test-case/volume-basic-test.sh

```diff
@@ -36,13 +36,19 @@ func_opt_parse_option "$@"
 setup_kernel_check_point
 tplg=${OPT_VAL['t']}
 maxloop=${OPT_VAL['l']}
-
+: "${SOF_ALSA_TOOL:="alsa"}"
 start_test
 
 func_error_exit()
 {
     dloge "$*"
-    pkill -9 aplay
+    if [[ "$SOF_ALSA_TOOL" = "alsa" ]]; then
+        pkill -9 aplay
+    elif [[ "$SOF_ALSA_TOOL" = "tinyalsa" ]]; then
+	pkill -9 tinyplay
+    else
+	echo "Unknowns asla tool $SOF_ALSA_TOOL"
```

Comment:
indent + typo

This fallback does not seem necessary. This is a clean up function, if SOF_ALSA_TOOL is wrong then there must have been another error message already printed much earlier than this.